src/lib/types/model.ts

```diff
@@ -402,6 +402,8 @@ export interface ISegment {
     description?: string;
     project?: string;
     constraints: IConstraint[];
+    usedInProjects?: number;
```

Comment:
does it have to be optional? Is there any scenario where we don't know the number?

Reply:
Yes, only case when we know the number is the when we load the list. Throughout app, other places, we dont know it about segment